### PR TITLE
Increase spec coverage

### PR DESCRIPTION
**Pull request by dylan-harper**
# Description

Please check the type of change your PR introduces:
- [ ] Bugfix (change which fixes an issue)
- [ ] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

I went and added some validations for creating a story that ensure the proper keys/values are included in storyCreate. Added some new tests, would love some eyes to edit, pull down, add, etc. 

 

### Type of change

Please check the relevant option.

- [x] Non-Breaking Change
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Big Breaking change (this breaks everything and I am sorry)

# Where should reviewers start?

Check out the story model and see what I was trying to do with the hash validations.

# How Has This Been Tested?

Please check the option(s) that best describe your tests. 

- [ ] New spec files were added
- [x] Tests were added to existing spec files
- [x] Some tests were changed
- [ ] No tests were changed
- [ ] All tests were changed (please explain what in God's name happened)

Added some new tests and query methods to buff out testing.

- Please add further details of your tests here.

# Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have removed all unused code or commented why I left it
- [ ] I have deployed my changes to a test-heroku and they work
- [x] My changes generate no new warnings
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: app/graphql/mutations/stories/update_story.rb [hunk 1]
```diff
@@ -13,7 +13,10 @@ class UpdateStory < Mutations::BaseMutation
 
       def resolve(attributes)
         begin
+          require "pry"; binding.pry
```
Comment (jbreit88) [inline app/graphql/mutations/stories/update_story.rb]: should probably get this pry out of here. I might even do a cmd+shft+f and see if there are any binding.pry hiding about in the code. We will pick all these blasted nits!

**Review by jbreit88 — APPROVED** [review]
Looks great! I like the addition of the testing, and thank you for covering the addition of average_words_per_minute. I totally forgot to add that to the tests.

**Review by JCNapier — APPROVED** [review]

**Comment by JCNapier** [comment]
This looks great Dylan, thank you for adding all of this testing. Solid work!

File: app/graphql/mutations/stories/update_story.rb [hunk 2]
```diff
@@ -14,16 +14,15 @@ class UpdateStory < Mutations::BaseMutation
       def resolve(attributes)
         begin
           story = Story.find(attributes[:id])
+
+          #add line for validating image and sound?
```
Comment (kevingloss) [inline app/graphql/mutations/stories/update_story.rb:18]: Should this comment be removed?

File: config/initializers/cors.rb [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins '*'
+    origins 'localhost:3000', 'localhost:4000', 'https://one-minute-writer.herokuapp.com'
```
Comment (kevingloss) [inline config/initializers/cors.rb:10]: Do we need an origin for the Python heroku?

**Review by kevingloss — APPROVED** [review]
Two comments that are more a quick check, but otherwise looks great!

File: app/graphql/mutations/stories/update_story.rb (hunk 2 above)
Comment (dylan-harper) [reply app/graphql/mutations/stories/update_story.rb:18]: fixed!

File: config/initializers/cors.rb (hunk 3 above)
Comment (dylan-harper) [reply config/initializers/cors.rb:10]: Don't think so, Python might need ours however...

**Pull request merged**